plugins/sentinelone/plugin.spec.yaml

```
@@ -3044,6 +3044,29 @@ actions:
         type: integer
         required: true
         example: 1
+  move_between_sites:
+    title: Move Agent to Another Site
+    description: Move an agent to another Site
+    input:
+      targetSiteId:
+        title: Target Site ID
+        description: A id of the new site to move the agents to
```

```suggestion
        description: A ID of the new site to move the agents to
```

---

Just to follow up on this one
```suggestion
        description: The ID of the new site to move the agents to
```

---

This has been updated